exec/jdbc-all/pom.xml

```diff
@@ -572,7 +572,7 @@
                           This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
 
                         </message>
-                        <maxsize>29000000</maxsize>
+                        <maxsize>31000000</maxsize>
```

Comment:
I played with this branch and I have to change the size of 31000000 to 32000000, it might due to my build environment but we may want to increase it to 32000000.